nav2_experimental_rl/turtlebot3_env.py

```diff
@@ -138,11 +140,12 @@ def reset(self):
         while not self.reset_simulation.wait_for_service(timeout_sec=1.0):
             print('Reset simulation service is not available...')
         self.reset_simulation.call_async(Empty.Request())
-        sleep(2)
+
         self.laser_scan_range = [0] * 360
         self.states_input = [3.5] * 8
+        while not self.scan_msg_received and rclpy.ok():
```

Comment:
while true looks need some throttling or this is going to consume a core while waiting

Reply:
Fixed.